File: Practice Sorting/7_Simple Statistics.py
t = int(input())
for _ in range(t):
    n, k = map(int, input().split())
    a = list(map(int, input().split()))
    # your code goes here
    a.sort()
    running_sum=0
    for i in range(k,n-k):
        running_sum+=a[i]
    print(running_sum/((n-2*k)))

File: Practice Sorting/codigo_cs/7Simple Statistics.cs
using System;

public class Test7
{
    public static void SimpleStatistics()
    {
        // your code goes here

        int t = int.Parse(Console.ReadLine());
        for (int i = 0; i < t; i++)
        {

            string input = Console.ReadLine();
            int n = int.Parse(input.Split(' ')[0]);
            int k = int.Parse(input.Split(' ')[1]);
            int[] arr = Array.ConvertAll(Console.ReadLine().Split(' '), int.Parse);
            // your code goes here
            Array.Sort(arr);
            int sum = 0;
            for (int j = k; j < n - k; j++)
            {
                sum += arr[j];
            }
            double average = (double)sum / (n - 2 * k);
            Console.WriteLine(average.ToString("F6"));

        }
    }
}
